### PR TITLE
Tp #41 add like model

### PR DESCRIPTION
**Pull request by ning905**
- add Like model and build relationships with the User and Post
- modify seed.js to create two likes in the DB
- include likes data when server responses with post data

### CODE REVIEW

**Review by miresque — APPROVED** [review]

File: prisma/schema.prisma [hunk 1]
```diff
@@ -70,3 +72,14 @@ model DeliveryLogLine {
   logId   Int
   log     DeliveryLog @relation(fields: [logId], references: [id])
 }
+
+model Like {
+  likedBy     User     @relation(fields: [likedById], references: [id], onDelete: Cascade)
+  likedById   Int
+  likedPost   Post     @relation(fields: [likedPostId], references: [id], onDelete: Cascade)
+  likedPostId Int
```
Comment (vherus) [inline prisma/schema.prisma:80]: I'm not a fan of these field names, they should represent the model they connect to.

```js
user       User
userId     Int
post       Post
postId     Int
```

With the above, a prisma query might look like

```js
await prisma.like.findMany({
    where: {
        userId: user.id
    }
})
```

Compare how easy to read that is with this:

```js
await prisma.like.findMany({
    where: {
        likedById: user.id
    }
})
```

**Review by vherus — CHANGES_REQUESTED** [review]

File: prisma/migrations/20220920112540_add_like_model/migration.sql [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+*/
+-- AlterTable
+ALTER TABLE "Profile" ALTER COLUMN "profileImageUrl" SET DEFAULT E'https://www.pngfind.com/pngs/m/676-6764065_default-profile-picture-transparent-hd-png-download.png';
+
+-- AlterTable
+ALTER TABLE "User" ALTER COLUMN "password" SET DATA TYPE VARCHAR(250);
+
+-- CreateTable
+CREATE TABLE "Like" (
+    "likedById" INTEGER NOT NULL,
```
Comment (Auenc) [inline prisma/migrations/20220920112540_add_like_model/migration.sql:15]: This migration file is a little odd to have now as we're adding columns, only for the next migration to remove them. Delete these two migration files and run the `npx prisma migrate dev` command to combine them

**Review by Auenc — CHANGES_REQUESTED** [review]
I think we can combine those two migration files

File: prisma/migrations/20220920112540_add_like_model/migration.sql (hunk 2 above)
Comment (ning905) [reply prisma/migrations/20220920112540_add_like_model/migration.sql]: Fixed!

**Review by Auenc — APPROVED** [review]
Looks good to me!

**Review by Noemi890 — APPROVED** [review]

**Pull request merged**